common4j/src/main/com/microsoft/identity/common/java/crypto/IDecryptor.java

```diff
@@ -49,4 +49,20 @@ byte[] decrypt(@NonNull final Key key,
                    @NonNull final String decryptAlgorithm,
                    final byte[] iv,
                    final byte[] dataToBeDecrypted) throws ClientException;
+
+    /**
+     * Decrypt the given byte array.
+     *
+     * @param key               the key to decrypt with.
+     * @param decryptAlgorithm  algorithm to decrypt with.
+     * @param iv                an initialization vector (IV).
+     * @param dataToBeDecrypted the data to be encrypted.
+     * @return a decrypted byte array.
+     */
+    byte[] decrypt(@NonNull final Key key,
```

Comment:
can name this differently to indicate it uses Aes Gcm.

Reply:
I've updated the name on the internal methods and kept the interface name unchanged.

Reply:
Alternatively, I can also refactor it to just take in parameter spec